File: Android/src/com/bu6ido/jsantase/notifications/ConsoleNotification.java
/**
 * 
 */
package com.bu6ido.jsantase.notifications;


/**
 * @author bu6ido
 *
 */
public class ConsoleNotification implements INotification 
{

	@Override
	public void info(String message) 
	{
		System.out.println(message);
	}

	@Override
	public void warning(String message) 
	{
		System.out.println("Warning: " + message);
	}

	@Override
	public void status(String message) 
	{
		System.out.println("Status -> " + message);
	}
}

File: Qt/console/console_notification.cpp
#include "console_notification.h"

ConsoleNotification::ConsoleNotification()
{
//  cout << "Constructing ConsoleNotification" << endl;
}

ConsoleNotification::~ConsoleNotification()
{
//  cout << "Destroying ConsoleNotifcation" << endl;
}

void ConsoleNotification::info(string message)
{
  cout << message << endl;
}

void ConsoleNotification::warning(string message)
{
  cout << "WARNING: " << message << endl;
}

void ConsoleNotification::status(string message)
{
  cout << "STATUS: " << message << endl;
}

